lib/web_ui/skwasm/image.cpp

```diff
@@ -100,7 +100,7 @@ class TextureSourceImageGenerator : public GrExternalTextureGenerator {
     glInfo.fTarget = GL_TEXTURE_2D;
```

Comment:
nit: also remove mipmapped on line 94 to make it more clear that param is not used in this impl

Reply:
Ah, good idea. Unfortunately this already merged. I'll see about cleaning this up in a followup.